guides/release/templates/input-helpers.md

```diff
@@ -31,13 +31,13 @@ helper:
   <tr><td>`selectionDirection`</td><td>`spellcheck`</td><td>`type`</td></tr>
 </table>
```

Comment:
This section can probably be deleted since you can pass literally any attributes

Reply:
Other than `@type`, `@value` and `@checked` (double check the API docs PR as the final source of truth). Could replace this with an example.